source/config/configSpec.py

```diff
@@ -68,12 +68,12 @@
 	cursorBlinkRate = integer(default=500,min=200,max=2000)
 	cursorShapeFocus = integer(default=192,min=1,max=255)
 	cursorShapeReview = integer(default=128,min=1,max=255)
-	noMessageTimeout = boolean(default=false)
+	# How braille display will show messages
+	# 0: Disabled, 1: Use timeout, 2: Show indefinitely
+	showMessages = integer(0, 4, default=1)
 	# Timeout after the message will disappear from braille display
-	# 0 means that no message will be shown
-	messageTimeout = integer(default=4,min=0,max=20)
-	tetherTo = string(default="focus")
-	autoTether = boolean(default=true)
+	messageTimeout = integer(default=4,min=1,max=20)
+	tetherTo = option("auto", "focus", "review", default="auto")

```

Comment:
could this get turned into an Enum? is a FeatureFlag more appropriate for this (or any other) of these?

Reply:
There are already `BrailleHandler.TETHER_*` values. I did not want to change this part of code just to convert it to an enum as it belongs to the interface and may be used by add-ons. Of course, 2023.1 is an API-breaking release; but IMO it's not worth modifying the API just for the beauty of the code.

Regarding featureFlags, if I understand the concept correctly, it's grossly an enum with a default value. It's adapted for  features in which we expect the default to be changed in future versions of NVDA (e.g. new features being tested). I do not imagine that "Tether to" option will be modified in the foreseeable future.

Let me know if you still want me to convert `BrailleHandler.TETHER_*` to an enum. Or if you just want me to introduce `BrailleHandler.TETHER_*` in the configSpec instead of hard-coded values (I do not know how).

Reply:
Done in faf36b3c204de23673bb2c46964f11cf641fd172.